src/bootstrap/builder.rs

```diff
@@ -1095,6 +1096,33 @@ impl<'a> Builder<'a> {
             rustflags.arg("-Zunstable-options");
         }
 
+        // not(bootstrap)
```

Comment:
Please use a regular attribute form here -- e.g., `// #[cfg(not(bootstrap)]`, that way it is easy to catch this when doing a project-wide grep during the [release process](https://forge.rust-lang.org/release/process.html#master-bootstrap-update-t-2-day-tuesday).

Reply:
That's what I tried to do, but apparently failed. :smile: 
Changed to `// #[cfg(not(bootstrap)]`, as suggested.